sycl/test-e2e/KernelFusion/cooperative_kernel.cpp

```diff
@@ -3,11 +3,6 @@
 
 // Test cooperative kernels are not fused
 
-// This test currently fails on AMD HIP due to an unresolved problem when
-// submitting a kernel with the use_root_sync property to a command group with
-// fusion enabled.
-// XFAIL: hip_amd
-
```

Comment:
This test is not failing anymore after changing hip program info returned for program device to return context device and not the binary associated device.